client/components/domains/map-domain-step/index.jsx

```diff
@@ -132,6 +133,17 @@ class MapDomainStep extends React.Component {
 					</div>
 
 					{ this.domainRegistrationUpsell() }
+
+					<div className="map-domain-step__domain-text">
+						{ translate(
+							"Update your domain's name servers to point it to your WordPress.com site. Domain registration and billing will remain at your current provider. {{a}}Learn more{{/a}}",
```

Comment:
Mmmm, feels weird. The user hasn't mapped the domain, and it says, `Update your...`. But maybe I'm overthinking. Would probably go with `You'll need to update...`

Reply:
Personally, I think "Update your..." is written more in a technical format. "You'll need to update..." sounds a bit passive, but I'll defer to @sixhours.

Reply:
The thing is, order should be:
- Map the domain first
- Then update the nameservers to point it to your wp.com site

Thanks for your input @jancavan.

Reply:
I was re-reading the heading and should we change it to - "Map **your** domain to use it as your site's address."

On the Domain Transfer page, the copy around it is more descriptive (of what's going to generally happen), but on Domain Mapping, it's more instructional. 

What does everyone think?

Reply:
@Automattic/cobalt would love you bunch chime in. :)

Reply:
How about something more like:
> **Requires** updating your domain's name servers to point it to your WordPress.com site.

Then we keep the timeline vague on purpose, but set the requirements explicitly (which is the aim here as I understand).